Sources/Helper/Array+Safe.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+extension Array {
+    public subscript(index: Int, default defaultValue: @autoclosure () -> Element) -> Element {
```

Comment:
Oops, I forgot to tell you, this extension is already defined in here (I believe Helper already depends on FoundationExtensions):

https://github.com/teufelaudio/FoundationExtensions/blob/master/Sources/FoundationExtensions/Collection/Collection%2BExtensions.swift#L77

There's no `defaultValue` parameter, but a simple `??` operator would solve that.

Reply:
alright. thanks. Some handy tooling